packages/dev-frontend/src/pages/Dashboard.tsx

```diff
@@ -18,21 +18,33 @@ export const Dashboard = (): JSX.Element => (
       </Heading>
       <Container variant="dashboardGrid">
         <Container variant="oneThird">
-          <PriceCard />
+          <RedemptionPriceCard />
         </Container>
         <Container variant="oneThird">
-          <BorrowingFee />
+          <OraclePriceCard />
         </Container>
         <Container variant="oneThird">
-          <OpenedVaults />
+          <MarketPriceCard />
         </Container>
         <Container variant="oneThird">
-          <ColRatio />
+          <RedemptionRateCard />
         </Container>
-        <Container variant="twoThirds">
-          <Chart />
+        <Container variant="oneThird">
+          <DeviationFactorCard />
         </Container>
+        {/* <Container variant="oneThird">
+          <BorrowingFee />
+        </Container> */}
+        {/* <Container variant="oneThird">
```

Comment:
Are these still meant to be commented out?

Reply:
Yeah, these are some less relevant stats that I removed as it looked a bit crowded.